src/cbc_sdk/enterprise_edr/base.py

```diff
@@ -197,20 +197,19 @@ def process_pids(self):
             return None
 
 
-class Event(UnrefreshableModel):
-    """Events can be queried for via `CBCloudAPI.select` or an already selected process with `Process.events`."""
-    urlobject = '/api/investigate/v2/orgs/{}/events/{}/_search'
-    validation_url = '/api/investigate/v1/orgs/{}/events/search_validation'
-    default_sort = 'last_update desc'
-    primary_key = "process_guid"
+class EnrichedEvent(UnrefreshableModel):
```

Comment:
This is an endpoint_standard model not enterprise_edr

Reply:
Changed to endpoint_standard.

Reply:
Changed as requested